lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -51,8 +51,17 @@ class TraceOfTab extends ComputedArtifact {
 
     // The first TracingStartedInPage in the trace is definitely our renderer thread of interest
     // Beware: the tracingStartedInPage event can appear slightly after a navigationStart
-    const startedInPageEvt = keyEvents.find(e => e.name === 'TracingStartedInPage');
+    const startedInPageEvt = keyEvents.find(e => {
+      if (e.name !== 'TracingStartedInPage') return false;
+      // In the extension, the first TracingStartedInPage is on chrome-extension://…/logo-page.html
+      // We'll await the next, which is the page/frame of the target URL
+      const url = e.args.data && e.args.data.frames && e.args.data.frames[0] &&
+          e.args.data.frames[0].url;
+      return !(url.startsWith('chrome-extension:') || url.startsWith('data:'));
```

Comment:
can we limit this to our `logo-page.html` somehow? maybe with `/^chrome-extension:.*\/(logo|blank)-page.html$/`? It would be nice to keep intact the CLI use case of auditing chrome extension pages

Reply:
ya! sg

Reply:
I agree with Patrick that we could be a bit more specific here, we can even do `chrome-extension://${chrome.runtime.id}/logo-page.html`. Also, I don't quite understand why we exclude `data:` here because AFAIK only `chrome-extension:` was problematic. Was `data:` also showing up in the trace?